src/client/extension.ts

```diff
@@ -341,16 +352,19 @@ function handleWorkspaceFolderChange() {
     }
 }
 
-function showNotificationForCopilot(telemetry: TelemetryReporter) {
+function showNotificationForCopilot(telemetry: TelemetryReporter, telemetryData:string, countOfActivePortals: string) {
```

Comment:
nit: consider changing type of `countOfActivePorals` to be an integer or number to keep strong typing. Then convert to string only at the site needed.